docusaurus/docs/identities/enrolling.md

```diff
@@ -80,12 +80,13 @@ ziti edge enroll \
 ### 3rd Party CA - Auto
 
 When using a third party CA identity creation process in the Ziti Controller is
-automatic. The act of enrolling the identity will create it. Like "3rd Party CA
-- One Time Token" - this flow expects that a private key and certificate have
+automatic with this "auto" flow. The act of enrolling the identity will create it with the default role attributes that were specified when the 3rd party CA was federated.

```

Comment:
change federated to created since that's when the default role attributes would be established.